test/suite/helper_tiered.py

```diff
@@ -260,22 +260,44 @@ def tiered_conn_extensions(self, extlist):
         extlist.extension('storage_sources', self.ss_name + config)
 
     def download_objects(self, bucket_name, prefix):
-        import boto3
-        # The bucket from the storage source is expected to be a name and a region, separated by a 
-        # semi-colon. eg: 'abcd;ap-southeast-2'.
-        bucket_name, region = bucket_name.split(';')
-        
-        # Get the bucket resource and list the objects within that bucket that match the prefix for a
-        # given test.
-        s3 = boto3.resource('s3')
-        bucket = s3.Bucket(bucket_name)
-        objects = list(bucket.objects.filter(Prefix=prefix))
-
         # Create a directory within the test directory to download the objects to.
-        s3_object_files_path = 's3_objects/'
-        if not os.path.exists(s3_object_files_path):
-            os.makedirs(s3_object_files_path)
+        object_files_path = 'objects/'
+        if not os.path.exists(object_files_path):
+            os.makedirs(object_files_path)
+
+        if (self.ss_name == 's3_store'):
+            import boto3
+            # The bucket from the storage source is expected to be a name and a region, separated by a 
+            # semi-colon. eg: 'abcd;ap-southeast-2'.
+            bucket_name, region = bucket_name.split(';')
+            
+            # Get the bucket resource and list the objects within that bucket that match the prefix for a
+            # given test.
+            s3 = boto3.resource('s3')
+            bucket = s3.Bucket(bucket_name)
+            objects = list(bucket.objects.filter(Prefix=prefix))
+
+            for o in objects:
+                file_path = object_files_path + '/' + o.key.split('/')[-1]
+                bucket.download_file(o.key, file_path)
+        elif (self.ss_name == 'gcp_store'):
+            from google.cloud import storage
+            
+            storage_client = storage.Client()
+            bucket = storage_client.bucket(bucket_name)
```

Comment:
We don't use `bucket` for anything - should we?

Reply:
Oops! We don't need that line removed now [22b35ac](https://github.com/wiredtiger/wiredtiger/pull/8953/commits/22b35acd1575bfed2e228baece7e3d1775026f83)